soc/xtensa/intel_adsp/ace/include/intel_ace15_mtpm/adsp_shim.h

```diff
@@ -6,50 +6,68 @@
 #ifndef ZEPHYR_SOC_INTEL_ADSP_SHIM_H_
 #define ZEPHYR_SOC_INTEL_ADSP_SHIM_H_
 
-/* The "shim" block contains most of the general system control
- * registers on cAVS platforms.  While the base address changes, it
- * has remained largely, but not perfectly, compatible between
- * versions.
+#ifndef _ASMLANGUAGE
+
+/**
+ * DfPMCCU
```

Comment:
name in comment does not match struct name

Reply:
@kfrydryx please re-review and resolve the conversation if it is o.k for you ;)